lib/auth/grpcserver.go

```diff
@@ -3263,6 +3263,8 @@ func (g *GRPCServer) GetClusterNetworkingConfig(ctx context.Context, _ *emptypb.
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
+	// NOTE: the cnc value returned by this method *must not* be mutated as it may be
+	// a shared reference that other goroutines may be reading.
```

Comment:
I am not sure I follow this comment. Aren't we always doing a Clone?

Reply:
Accidental holdover from an earlier iteration of this feature that still allowed escaping the readonly interface.  Removed.